project/AkkaBuild.scala

```diff
@@ -1428,11 +1458,17 @@ object AkkaBuild extends Build {
       "com.google.protobuf")
 
     def exports(packages: Seq[String] = Seq(), imports: Seq[String] = Nil) = osgiSettings ++ Seq(
-      OsgiKeys.importPackage := imports ++ defaultImports,
+      OsgiKeys.importPackage := imports,
+      OsgiKeys.importPackage <++= scalaVersion { v =>
```

Comment:
Are you sure this change is at the right place? This will add `scala.xml` to any package that uses `exports`, right?